docs/docs/site-docs.js

```diff
@@ -83778,7 +83780,7 @@
 /* 665 */
 /***/ (function(module, exports) {
 
-	module.exports = [{"name":"@blueprintjs/core","version":"1.32.0"},{"name":"@blueprintjs/datetime","version":"1.23.0"},{"name":"@blueprintjs/docs","version":"1.1.1"},{"name":"@blueprintjs/labs","version":"0.12.0"},{"name":"@blueprintjs/table","version":"1.29.0"}]
+	module.exports = [{"name":"@blueprintjs/core","version":"1.32.0"},{"name":"@blueprintjs/datetime","version":"1.23.0"},{"name":"@blueprintjs/docs","version":"1.1.1"},{"name":"@blueprintjs/labs","version":"0.12.0"},{"name":"@blueprintjs/table","version":"1.29.1"}]
```

Comment:
this file should not be changed here, only in release PRs. can you revert these changes?